### ISSUE TITLE
Unable to access the hud bar with VoiceOver.

### ISSUE THREAD

**Issue by diegov12** [OpenEmu/OpenEmu#4389]
I'm currently using 2.3 on catalina, and I'm unable to get it to show up in any game.
